app/public/js/player/playerCtrl.js

```diff
@@ -289,6 +289,16 @@ app.controller('PlayerCtrl', function (
     });
 
     hotkeys.add({
+        combo: ['command+h', 'ctrl+h'],
+        description: 'Heart track',
+        callback: function() {
+          $timeout(function() {
```

Comment:
Hm I checked not sure why we have that timeout over there.. please remove and test again if works just leave the $timeout out and I will merge.. thanks